### DISCUSSION TITLE
Extend ContentmentContentContext to access parentId for v4

### DISCUSSION THREAD

**Discussion by gordonsaxby** [leekelleher/umbraco-contentment#415] [Question] [answered]
Is this fix only for v5 https://github.com/leekelleher/umbraco-contentment/pull/389

It would be really useful if it could be also applied to v4. Unfortunately we are not in a position to upgrade the affected Umbraco v10 site to v13 just yet.

**Comment by leekelleher** [comment, marked as answer]
My current focus (in my free time, for open-source work) is Contentment v6 (to support Umbraco v14+).

I'd be open to a pull request to backport this particular fix for v4, (targeting the `dev/v4.x` branch) and I'd be happy to put out a v4.7.1 release. But it would need to be tested against Umbraco v8, v10 and v13.

**Reply by gordonsaxby** [reply]
Thanks for your quick reply. Unfortunately I don't personally have the time to do that. We have implemented in way that does not require the use of Nested Content or Block List.
It is very handy to know that it's been fixed in v5 though (for future / other projects)
